packages/rollup-plugin-invariant/src/plugin.ts

```diff
@@ -18,10 +18,11 @@ export default function invariantPlugin(options = {} as any) {
       recast.visit(ast, {
         visitImportDeclaration(path) {
           this.traverse(path);
-          const node = path.value;
+          const node = path.node;
           if (
             needToImportProcess &&
             node.source.value === "ts-invariant" &&
+            node.specifiers &&
             !node.specifiers.some((spec: any) => {
```

Comment:
Now that `node.specifiers` is typed as `recast.types.namedTypes.ImportDeclaration` instead of `any`, TypeScript was able to warn that `node.specifiers` might be undefined here.